pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -193,6 +195,10 @@ type TidbClusterSpec struct {
 	// +optional
 	PD *PDSpec `json:"pd,omitempty"`
 
+	// PD cluster spec
+	// +optional
+	TSO *PDSpec `json:"tso,omitempty"`
```

Comment:
`TSO cluster spec`?

BTW, will we name it as _TSO_ directly or something like _PD TSO_?

Reply:
Will all fields in `PDSpec` be used in `TSO`? if not, how about adding some comments to those fields?
also for `TSOStatus`

Reply:
provide PDMS spec